README.md

```diff
@@ -1,194 +1,122 @@
 
-Lastly, you can just use `unwrap`, which is less safe because it will throw an error.
-
-```typescript
-try {
-  // Possible error is thrown here and it contains human readable description of what went wrong :)
-  const value = result.unwrap();
-  // You can accees value here
-  console.log(value);
-} catch (e) {
-  console.log(e);
-}
-```
+Check the [Superface Catalog](https://superface.ai/catalog) for existing use-cases or learn how to [write your own use-cases](https://superface.ai/docs/guides/how-to-create).
 
-## Configuration
+<!-- TODO: point to docs for working with the result object -->
 
-The Superface OneSDK is configurable through various environment variables:
- - `SUPERFACE_SDK_TOKEN` - your auth token to integrate your running instance with Superface services
- - `SUPERFACE_API_URL` - URL of the Superface services, you probably don't need to change this; default is https://superface.ai
- - `SUPERFACE_PATH` - path to your super.json file; default is `./superface/super.json`
- - `SUPERFACE_METRIC_DEBOUNCE_TIME_MIN` and `SUPERFACE_METRIC_DEBOUNCE_TIME_MAX` - to rate limit metric reporting, OneSDK will send aggregated metrics after at least `MIN` milliseconds and at most `MAX` milliseconds; default is 1000 for min and 60000 for max
- - `SUPERFACE_DISABLE_METRIC_REPORTING` - set this variable to disable metric reporting; enabled by default
+## Support
 
-## Metric reporting
+If you have any questions, want to report a bug, request a feature or you just want to talk, feel free to [open an issue](https://github.com/superfaceai/one-sdk-js/issues/new/choose) or hop on our [Discord server](https://sfc.is/discord).
 
-The Superface OneSDK will send info about usage to Superface services. This info is anonymized, rate limited and allows you to see how the client is performing on your dashboard. To be able to see those metrics, you need to provide your auth token.
-There are three kinds of metrics reported at present - one is sent when the client instance is created, one after each perform (reporting success or failure) and one when a provider failover happens. The reports can be disabled or configured with [environment variables](#configuration).
-For metrics to be successfuly sent, the application needs to be properly exited, i.e. there should be no unhandled Promise rejections or exceptions.
+You can find more options for reaching us on the [Support page](https://superface.ai/support).
 
 ## Security
 
-Superface is not man-in-the-middle so it does not require any access to secrets that are needed to communicate with provider API. Superface SDK only reads super.json file, resolved authorization secrets from environment variables or from the file itself and applies them to network requests as required by the specific map.
+Superface doesn't act as a proxy and communicates with API providers directly; the secrets are sent only to the providers (see [how Superface works](https://superface.ai/how-it-works)).
```

Comment:
Superface is not a proxy. The calls are always going directly from your application to API providers. The API secrets are never sent anywhere else but to the used provider's API ...